solutions/usaco-833.mdx

```diff
@@ -2,13 +2,105 @@
 id: usaco-833
 source: USACO Bronze 2018 Open
 title: Family Tree
-author: Danh Ta Chi Thanh
+author: Danh Ta Chi Thanh, Ryan Chou
 ---
 
 ## Solution
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_family_bronze_open18.html)
```

Comment:
since your solution is different from the alternative solution, put it in a separate languagesection